examples/src/main/AndroidManifest.xml

```diff
@@ -96,8 +96,7 @@
     </activity>
 
     <activity
-        android:name="com.mapbox.navigation.examples.MainActivity"
-        android:screenOrientation="portrait">
```

Comment:
Let's add a separate activity. Also, why not add it to the QA module since `exmaples` goes away anyway?

Reply:
The decision was made not to add any test activity to QA test app.